backend/tests/equipment_controller_test.js

```diff
@@ -18,32 +18,32 @@ describe("Equipment Controller", function() {
             const mockMethod = sinon.stub(equipment, "findAll");
             mockMethod.returns("Method Find All was Successfully Called");
 
-            toCheck = await EquipmentController.get(request);
+            let toCheck = await EquipmentController.get(request);
             expect(toCheck).to.be.equal("Method Find All was Successfully Called");
         });
     });
 
-    // describe("Testing Post Function", async function() {
-    //     var request = {
-    //         body: {
-    //             data: {
-    //                 name: null
-    //             }
-    //         }
-    //     };
+    describe("Testing Post Function", async function() {
+        var request = {
+            body: JSON.stringify({
+                name: null
+            })
+        };
 
-    //     it("Should Call Nothing Due to Bad Request", async function() {
-    //         toCheck = await EquipmentController.post(request);
-    //         expect(toCheck).to.be.equal(undefined);
-    //     });
+        it("Should Call Nothing Due to Bad Request", async function() {
+            toCheck = await EquipmentController.post(request);
+            expect(toCheck).to.be.equal(undefined);
+        });
     
-    //     it("Should Call Create Method", async function() {
-    //         const mockMethod = sinon.stub(equipment, "create");
-    //         mockMethod.returns("Create Method was Successfully Called");
-    //         request.body.name = "username";
+        it("Should Call Create Method", async function() {
+            const mockMethod = sinon.stub(equipment, "create");
+            mockMethod.returns("Create Method was Successfully Called");
+            request.body = JSON.stringify({
+                name: "username"
+            })
 
-    //         toCheck = await EquipmentController.post(request);
-    //         expect(toCheck).to.be.equal("Create Method was Successfully Called");
-    //     });
-    // });
+            toCheck = await EquipmentController.post(request);
```

Comment:
in other files toCheck is redeclared with let each time it is  used. Either do it that way, or  use the same variable as here, but do it consistently